book/src/drivers.md

```diff
@@ -33,6 +33,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [IBM Block Storage](https://github.com/ibm/ibm-block-csi-driver) | `block.csi.ibm.com` | v1.0, v1.1 | A Container Storage Interface (CSI) [Driver](https://www.ibm.com/support/knowledgecenter/SSRQ8T) for IBM FlashSystem 9100, IBM Spectrum Virtualize, IBM Storwize, IBM FlashSystem A9000 and A9000R. | Persistent | Read/Write Single Pod | Yes | 
 [Intel PMEM-CSI](https://github.com/intel/pmem-CSI) | `pmem-csi.intel.com` | v1.0 | A Container Storage Interface (CSI) driver for PMEM from Intel | Persistent | Read/Write Single Pod | Yes | Raw Block
 [JuiceFS](https://github.com/juicedata/juicefs-csi-driver) | `csi.juicefs.com` | v0.3, v1.0 | A Container Storage Interface (CSI) Driver for JuiceFS File System | Persistent | Read/Write Multiple Pod | Yes | 
+[kaDalu](https://github.com/kadalu/kadalu) | `org.kadalu.gluster` | v0.3 | A CSI Driver (and operator) for GlusterFS | Persistent | Read/Write Multiple Pods | Yes | Block, FileSystem
```

Comment:
For the last column, we only allow "Raw Block" as an additional feature to be listed.

Reply:
Corrected.